platforms/posix/src/main.cpp

```diff
@@ -506,24 +531,26 @@ void print_usage()
 {
 	printf("Usage for Server/daemon process: \n");
 	printf("\n");
-	printf("    px4 [-h|-d] [-s <startup_file>] [-d <test_data_directory>] [<rootfs_directory>]\n");
+	printf("    px4 [-h|-d] [-s <startup_file>] [-d <test_data_directory>] [<rootfs_directory>] [-i <instance>]\n");
```

Comment:
Hang on? So we have `-d` for daemon mode, and we have `-d <dir>` for the test data directory?

And shouldn't there be a `-i <instance>` in the printf?

Reply:
Right, it's just a documentation fail, it should be `-t`. I'll fix that.